docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:latest as builder
```

Comment:
@Ompragash are we using latest lags or last known stable?

Reply:
yup, it was tagged to stable until this [commit](https://github.com/harness-community/drone-findfiles/pull/1/commits/2fc48894cc820dae739b26522d00877b32955d09). @Fernando-Dourado please update the tag pointing to `3.19.4`

Reply:
Done